.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+
+on:
+  # publish from the Releases page:
+  release:
+    types: [published]
+  # publish from the Actions page:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (e.g. 2.0.3)'
```

Comment:
oh snap, good catch.

[PEP440 explicitly supports dates](https://www.python.org/dev/peps/pep-0440/#support-for-date-based-version-identifiers), but under duress, I think. I think it would be good to move away from them since our software doesn't use them. Something to discuss with the rest of the team maybe?

Reply:
I'm not totally sure this works like I think. When you click it you get this box:

![Screenshot from 2021-04-11 03-51-11](https://user-images.githubusercontent.com/987487/114296397-3d97a200-9a79-11eb-8f96-15f3cd6a43ed.png)

If you change the branch will it tag *that one*? We need to confirm which way this goes that before committing to it.

Reply:
Oh, that's interesting. I'm not sure!

(When I was testing `workflow_dispatch` for the changelog PR, my personal repo only had a single branch, so I haven't run into this yet.)